### PR TITLE
[DFFE-7] Style the Product Details Page using CSS

### PR DESCRIPTION
**Pull request by DeeDeji20**
Web capture_1-1-2021_1544_
Web capture_1-1-2021_15522_
Web capture_1-1-2021_15441_

### CODE REVIEW

**Comment by adaugochi** [comment]
There is conflict on your PR, kindly resolve them

File: index.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                <a class="links" href="../html-forms/contact.html">Contact</a>
+            </li>
+        </ul>
+    </header>
+    <!-- <section class="description">
+        <div>
+            <h1>Clean & Classy</h1>
+            <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Doloremque amet quia iste itaque unde excepturi, veritatis temporibus voluptatem est non pariatur a sed incidunt optio obcaecati maxime nostrum commodi atque?</p>
+            <button class="explore-btn">Explore </button>
+        </div>
```
Comment (TundeOladejo) [inline index.html:37]: Remove these commented lines

File: index.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            <img src="./images/omen.jpg" alt="">
+            <div class="omen-descri">
+                <h3>HP Omen 15</h3>
+                <h4>Brand: HP</h4>
+                <label class="qty" for="qty">Quantity:</h4> <input type="text"><br>
+                <label class="price" for="price">Price:#550,000</label> <br>
+                <button class="buy-button btn">Buy now</button>
+            </div>
+        </div>
+        <div class="descript">
```
Comment (TundeOladejo) [inline index.html:52]: Your class name is not really cool. You might want to change it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                    <h3>Specifications</h3>
+                    <ul>
+                        <li>Intel Core i-7, 10th gen</li>
+                        <li>Hard Drive:512gb SSD</li>
+                        <li>Storage: 16gb RAM</li>
+                        <li>Size: 14"</li>
+                        <li>Color: Blue</li>
+                    </ul>
+                </div>
+            </div>
```
Comment (TundeOladejo) [inline index.html:88]: Format this block of code properly. I think you need to also remove one of those two div elements.

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

**Review by adaugochi — CHANGES_REQUESTED** [review]
You push people files along with your task. How can you have `426 File changed`

**Comment by Ireti203** [comment]
@DeeDeji20 Kindly resolve the conflicts on this branch

**Pull request closed without merging**